modules/install/wizard.php

```diff
@@ -4,8 +4,8 @@
     $db->success = 0;
 }
 $importXml = new \LanSuite\XML();
-$installImport = new \LanSuite\Module\Install\Import($importXml);
-$install = new \LanSuite\Module\Install\Install($installImport);
```

Comment:
Maybe I don't see it, but there is only the change of the variable name, right?
What does the Fatal error say? Or why will another name fix it?

Reply:
Ok. Saw it here. Sorry: https://github.com/lansuite/lansuite/issues/440